rs/ledger_suite/icrc1/tokens_u256/src/lib.rs

```diff
@@ -12,6 +13,8 @@ use serde::{
 use std::borrow::Cow;
 use std::fmt;
 
+pub mod encode_decode;
```

Comment:
could we use a more descriptive name, e.g., minicbor_u256?

Reply:
Done!